subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -842,6 +842,13 @@ uint8_t ll_rx_get(void **node_rx, uint16_t *handle)
 
 	*node_rx = NULL;
 
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
+	cmplt = ull_iso_tx_ack_get(handle);
+	if (cmplt) {
+		return cmplt;
+	}
+#endif /* CONFIG_BT_CTLR_ADV_ISO || CONFIG_BT_CTLR_CONN_ISO */
```

Comment:
Don't we need to make sure that only the last PDU counts? I assume that LLID in BIS PDU indicates if this is the last one. Otherwise one SDU could generate more than one complete.

Reply:
In this PR without ISOAL Tx module, one SDU = one PDU. But I agree, unless ISOAL Tx needs to know each PDU which I guess so for framed PDUs (and may be for unframed)?, only END PDUs could generate the num complete.